cmd/kubeadm/app/cmd/version.go

```diff
@@ -35,10 +44,39 @@ func NewCmdVersion(out io.Writer) *cobra.Command {
 			kubeadmutil.CheckErr(err)
 		},
 	}
+	cmd.Flags().BoolP("short", "", false, "only print the version (tag|number)")
+	cmd.Flags().StringP("output", "o", "", "output format, options available are yaml and json")
```

Comment:
didn't check how kubectl does it, but to be `-o short` would be better than two flags